File: src/main/java/org/manapart/enderports/ItemGroupEP.java
package org.manapart.enderports;

import net.minecraft.item.ItemGroup;
import net.minecraft.item.ItemStack;

public class ItemGroupEP extends ItemGroup {
    public static final ItemGroupEP instance = new ItemGroupEP(ItemGroup.getGroupCountSafe(), "enderports");

    private ItemGroupEP(int index, String label) {
        super(index, label);
    }

    @Override
    public ItemStack makeIcon() {
        return new ItemStack(EnderPorts.enderportsIcon);
    }

}

File: src/main/java/org/manapart/enderports/ItemGroupEP.kt
package org.manapart.enderports

import net.minecraft.world.item.CreativeModeTab
import net.minecraft.world.item.ItemStack

class ItemGroupEP private constructor(index: Int, label: String) : CreativeModeTab(index, label) {
    override fun makeIcon() = ItemStack(EnderPorts.icon)

    companion object {
        val instance = ItemGroupEP(getGroupCountSafe(), "enderports")
    }
}